Misc/NEWS.d/next/Library/2024-05-09-12-33-25.gh-issue-118827.JrzHz1.rst

```diff
@@ -0,0 +1,2 @@
+Remove deprecated ``Quoter`` class from :mod:`urllib.parse`. It had
+previously raised a :exc:`DeprecationWarning` since Python 3.11.
```

Comment:
```suggestion
Remove deprecated :class:`!Quoter` class from :mod:`urllib.parse`. It had
previously raised a :exc:`DeprecationWarning` since Python 3.11.
Patch by Nikita Sobolev.
```